apps/block_scout_web/lib/block_scout_web/graphql/celo/schema/query_fields.ex

```diff
@@ -11,7 +11,7 @@ defmodule BlockScoutWeb.GraphQL.Celo.QueryFields do
   defmacro generate do
     quote do
       @desc "Gets token transfer transactions."
-      connection field(:token_transfer_txs, node_type: :transfer_tx) do
+      connection field(:token_transfer_transactions, node_type: :transfer_transaction) do
```

Comment:
```suggestion
      connection field(:token_transfer_txs, node_type: :transfer_transaction) do
```

Unfortunatelly, we cannot change the name of this connection field, otherwise we lose backward compatibility with Celo fork of Blockscout, which is required by Valora client.